contrib/python/tests/python/pants_test/contrib/python/checks/checker/test_pyflakes.py

```diff
@@ -25,10 +25,6 @@ def test_pyflakes_unused_import(self):
       # rather than with Pants.
       self.assertNit('import os', 'F401', expected_line_number='001-002')
 
-  def test_pyflakes_ignore(self):
```

Comment:
Why delete this?

Reply:
Because it was shadowed by another function with the same name (see commit message)